src/style.css

```diff
@@ -0,0 +1,81 @@
+@font-face {
+  font-family: "Avenir";
+  font-weight: 300;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+body {
+  margin: 0;
+}
+
+.form-field {
+  position: relative;
+  display: block;
+  margin: 20px 8px;
+}
+
+input {
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  border-radius: 4px;
+  font-family: "Avenir", sans-serif;
+}
+
+input:hover {
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+  font-family: "Avenir", sans-serif;
+}
+
+input:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  border-radius: 3px;
+  font-family: "Avenir-Heavy", sans-serif;
+}
+
+.big {
+  width: 100%;
+  padding: 26px 7px 24px;
+  text-indent: 55px;
+}
+
+.small {
+  width: 100%;
+  padding: 12px 11px 10px;
+  text-indent: 22px;
+}
+
+.bigLoupe {
+  position: absolute;
+  height: 19px;
+  width: 19px;
+  top: 25px;
+  left: 26px;
+}
+
+.smallLoupe {
+  position: absolute;
+  height: 11px;
+  width: 11px;
+  top: 15px;
+  left: 13px;
+}
+
+.big::placeholder {
+  font-size: 16px;
+  color: #3d4e61;
+}
+
+.small::placeholder {
+  font-size: 14px;
+  color: #3d4e61;
+}
```

Comment:
You duplicate styles. [Checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md) the 2nd item